portal/src/main/java/org/bonitasoft/web/toolkit/client/ClientApplicationURL.java

```diff
@@ -209,6 +209,11 @@ public static void setPageToken(final String pageToken, final boolean refresh) {
         self._setPageToken(pageToken, refresh);
     }
 
+    public static void initPageToken(final String token) {
+        self.attributes.addValue(ATTRIBUTE_TOKEN, token);
+        Window.Location.replace("?" + UrlSerializer.serialize(self.attributes));
```

Comment:
I think it should rather be : '#?' instead of just '?'.
I you use ? it will replace the query string if there is on (for example the tenant id could be removed)